### ISSUE TITLE
real-time translation no longer works

### ISSUE THREAD

**Issue by scythescope** [FilipePS/Traduzir-paginas-web#470]
Add-on used to work fine for translating subtitles of a video in real-time. Now it only translates the web page (along with the subtitles) once I click on it but that only lasts for a while until another subtitle comes on and replaces the old one. This never used to happen before, I don't know what changed. Is anyone else experiencing the same problem? Any guidance will be appreciated.

**Comment by taoshengqiansui** [comment]
me the same

**Comment by FilipePS** [comment]
Subtitles on YouTube? YouTube itself has an option to translate the subtitles.

**Comment by scythescope** [comment]
> 

no, not on youtube. it's also a streaming platform, though. it never used to happen before is the issue.
